src/cascadia/TerminalApp/TabBase.cpp

```diff
@@ -388,6 +379,25 @@ namespace winrt::TerminalApp::implementation
             subtleFillColorTertiaryBrush.Color(subtleFillColorTertiary);
         }
 
+        // The tab font should be based on the evaluated appearance of the tab color layered on tab row.
+        const auto layeredTabColor = til::color{ color }.layer_over(_tabRowColor);

```

Comment:
What does this do if the `_tabRowColor` is fully transparent?

Reply:
oooooh good question

Reply:
I'd _guess_ that it would treat it as black, cause `alpha=0 * (r,g,b) => (0,0,0)`

Reply:
Yep, we treat it as 0,0,0 and end up with white text.